### ISSUE TITLE
Another issue related to campus search bar and drop-down menu listing

### ISSUE THREAD

**Issue by Ruth-DeTro** [woodwarddesign/ccc-transfer#195]
Nothing is working on this menu/page. When first loaded one can see all campuses, but then select anything from the drop-down listing, be it "all", a segment (CSU, UC, AICCU, OOS) or a state nothing appears - not a campus to be seen.  Ray is presenting the website today and tomorrow to the Statewide Transfer Directors Meeting and all needs to work properly. Please advise.

**Comment by mlwoodward** [comment]
Fixed.  Will look into making all states active.  I believe this is tied to an issue with WYSIWYG editor (ckeditor).  I am disabling this tool to test further.

**Comment by mlwoodward** [comment]
Issue number #196 has been merged into this issue.

**Comment by mlwoodward** [comment]
All states have been added back into the system.  I've also created a backup of the code snippet in the block if this comes back.  The block appears to be trying to strip out the data-filter tag that causes the listing to filter correctly.  As long as no one goes into the edit view of that block, it should continue to work correctly.  For now I am going to close this task.  

**Note to self**:  If we need to find the code snippet, it can be found here - prometheus/state.php
